### PR TITLE
Fixes #42 button problem. Also small tweak to html style.

### LINKED ISSUE #42: After clicking search button it still looks depressed

### PR DESCRIPTION
**Pull request by bjedrzejewski**

### CODE REVIEW

**Comment by bjedrzejewski** [comment]
Dont merge yet

**Comment by bjedrzejewski** [comment]
Now this is good to merge.

File: public/index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
                         <br>
                         <br>
                         <input type="submit" class="btn btn-primary center-block" value="Search" 
-                               ng-disabled="searchCtrl.searchDisabled"/>
+                               ng-disabled="searchCtrl.searchDisabled" onclick="this.blur()"/>
                     </form>
```
Comment (owennw) [inline public/index.html]: A question revealing my ignorance here: is the use of `this` here safe?

Comment (bjedrzejewski) [inline public/index.html]: yes, this will refer to the dom element

**Pull request merged**